Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -245,6 +249,14 @@ private void ContentGrid_ManipulationStarted(object sender, ManipulationStartedR
                 if (_leftCommandPanel != null)
                 {
                     _leftCommandTransform = _leftCommandPanel.RenderTransform as CompositeTransform;
+
+                    _leftCommandAnimation = new DoubleAnimation();
```

Comment:
Regarding clipping...as this is not yet supported by the toolkit itself, I'm fine doing it using sb